renku/ui/cli/template.py

```diff
@@ -263,29 +264,36 @@ def update_template(force, interactive, dry_run):
     type=click.STRING,
     help="Git revision/branch/tag to validate the template at.",
 )
-def validate_template(ctx, json_format, revision):
+@click.option(
+    "--source",
+    type=click.STRING,
+    help="Remote template repository to clone and check.",
+)
+def validate_template(ctx, json_format, revision, source):
```

Comment:
To be in par with other `template` command (what Lorenzo said), we should rename `revision` to `reference` and also add `-r` and `-s`.